### PR TITLE
Downgraded tsconfig output to ES5, and package upgrades

### PR DESCRIPTION
**Pull request by thomastay**
Upgraded Webpack-Cli version and ts-loader to remove warnings.

Also, moved @types/lodash and ts-loader to dev-dependencies

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "compilerOptions": {
     "declaration": true,
     "noResolve": false,
-    "module": "esnext",
+    "module": "CommonJS",
```
Comment (masterkidan) [inline tsconfig.json:5]: Can you make all of this small cased, 
Like 
commonjs 
es5 ... etc ?

**Review by masterkidan — DISMISSED** [review]

File: tsconfig.json (hunk 1 above)
Comment (thomastay) [reply tsconfig.json]: fixed in latest commit

**Comment by thomastay** [comment]
@masterkidan it looks like the automated version bump failed?
```
remote: Permission to microsoft/ObjectStoreProvider.git denied to github-actions[bot].
fatal: unable to access '***github.com/microsoft/ObjectStoreProvider.git/': The requested URL returned error: 403
```

**Comment by masterkidan** [comment]
@thomastay:Thats because you are merging a fork, fork'd repos don't have access to github action perms of the parent (for security reasons). 
Can you try creating a branch on this repo itself and sending out the changes?

**Comment by masterkidan** [comment]
FYI: I added you as a member to this project so you should have access now.

**Comment by thomastay** [comment]
@masterkidan got it, didn't know I had access to push a branch. Will do so.

**Pull request closed without merging**